RecoLocalMuon/GEMSegment/plugins/ME0SegmentBuilder.cc

```diff
@@ -47,7 +47,9 @@ void ME0SegmentBuilder::build(const ME0RecHitCollection* recHits, ME0SegmentColl
     // save current ME0RecHit in vector associated to the reference id
```

Comment:
this will be subject of a forthcoming PR.  To comment to your earlier question:

> Also, in your slides
> https://indico.cern.ch/event/574385/contributions/2323764/attachments/1352593/2042703/20161011_PhaseIISim_ME0Status_Complete.pdf I see
>
>    urgent: Geometry, Digitizer
>    less urgent: Local Reco, Global Reco

> Given somewhat long history between PR open and first working version, it's unclear to me if all changes are needed minimally or you also have included the less urgent item

These are just the minimal changes: geometry + fixes to get the existing sim-digi-reco chain working in this new 2023D6 geometry. Forthcoming PRs will deal with an improved digitizer and improved segment (Dubna algorithm cfr CSC). For now we will leave the ME0Segment slightly suboptimal: a reduced number of hits (6->5,4) due to the search for rechits being limited to the same eta partition. Actually masking the rolls will increase too much the combinatorics ... I think we will try to add just the neighbouring rolls.